azurelinuxagent/common/osutil/iosxe.py

```diff
@@ -58,7 +59,7 @@ def publish_hostname(self, hostname):
         Restart NetworkManager first before publishing hostname
         """
         shellutil.run("service NetworkManager restart")
-        super(RedhatOSUtil, self).publish_hostname(hostname) # pylint: disable=E0602
+        super(IosxeOSUtil, self).publish_hostname(hostname)
```

Comment:
This seems like it was copy-pasted from Redhat. Anyone know if this is the case?

Reply:
Not sure, most of these distro-specific modules that are not endorsed came from external contributors.